src/catkin_pkg/package_templates.py

```diff
@@ -294,10 +294,12 @@ def _create_targetlib_args(package_template):
 
 
 def _create_include_macro(package_template):
-    result = '# include_directories(include)'
+    result = '#  include'
```

Comment:
Should this go into `includes` instead so that it gets indented the same way? Like this: `includes = ['# include']`

Nitpick: can you please remove one of the two spaces. The `#` should be followed by a single space.